.github/workflows/test-runner-on-perf.yml

```diff
@@ -62,6 +65,60 @@ jobs:
               fi
             fi
           done
+      - name: "Assemble data-repo code to generate version.properties file"
+        run: ./gradlew clean assemble
+      - name: "Read latest jade-data-repo gitHash from version.properties file"
+        id: read_property
+        uses: christian-draeger/read-properties@1.0.1
+        with:
+          path: './src/main/resources/version.properties'
+          property: 'gitHash'
+      - name: 'Checkout datarepo-helm-definitions repo'
+        uses: actions/checkout@v2
+        with:
+          repository: 'broadinstitute/datarepo-helm-definitions'
+          token: ${{ secrets.HELM_REPO_TOKEN }}
+          path: datarepo-helm-definitions
+      - name: "Update perf image tag with gitHash"
+        uses: docker://mikefarah/yq:latest
+        with:
+          args: yq w -i datarepo-helm-definitions/perf/datarepo-api.yaml image.tag ${{ steps.read_property.outputs.value }}
+      - name: "Create datarepo-helm-definition pull request with updated perf image tag"
+        uses: broadinstitute/create-pull-request@v3 # forked from peter-evans/create-pull-request
+        id: perfTagPR
+        with:
+          token: ${{ secrets.HELM_REPO_TOKEN }}
+          path: datarepo-helm-definitions
+          commit-message: "Perf Datarepo version update: ${{ steps.read_property.outputs.value }}"
+          committer: datarepo-bot <noreply@github.com>
+          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          title: "Perf Datarepo version update: ${{ steps.read_property.outputs.value }}"
+          branch: "version-update-${{ steps.read_property.outputs.value }}"
+          body: |
+            Update versions in perf env to reflect image tag ${{ steps.apiprevioustag.outputs.tag }}.
+            *Note: This PR was opened by the [update-env GitHub Actions workflow](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}).*
+          labels: "broadbot,datarepo,automerge"
+      - name: "Merge datarepo-helm-definition pull request"
+        run: |
+          cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-helm-definitions
+          git checkout master
+          git config pull.rebase false
+          if git rev-parse --verify origin/version-update-${{ steps.read_property.outputs.value }}; then
+            git merge origin/version-update-${{ steps.read_property.outputs.value }}
+            git branch -d version-update-${{ steps.read_property.outputs.value }}
+            git push
+          else
+            echo "Branch version-update-${{ steps.read_property.outputs.value }} not found."
+          fi
+          cd ${GITHUB_WORKSPACE}/${workingDir}
+      - name: "Setup helmfile"
+        uses: mamezou-tech/setup-helmfile@v0.5.0
```

Comment:
Before merging we should create our own `setup-helmfile` action for security reasons.

Reply:
Ok - I've forked setup-helmfile